tasks/src/main/java/io/main/FileService.java

```diff
@@ -41,33 +48,31 @@ private static int calculateAverageQtyOfFilesInFolder() {
   private static int calculateAverageLengthOfFileNames() {
     int nameFiles;
     int lengthNameFiles = 0;
-    for (String listFile : listOfFiles) {
+    for (String listFile : getListOfFiles()) {
       nameFiles = listFile.lastIndexOf('.');
       lengthNameFiles += nameFiles;
     }
     return lengthNameFiles / listOfFiles.size();
   }
 
-  static void getInfoAboutFile(Path path) throws IOException {
+  static void infoAboutFile(Path path) throws IOException {
```

Comment:
A method's name should express an action or intention meanly start with a verb - printFileInfo